client/src/pages/authentication/signin/index.js

```diff
@@ -79,12 +82,12 @@ const Signin = (props) => {
               error={errorMessage}
             />
             <Field
+              required          
```

Comment:
The `Field` component from `redux-form` doesn't take in `required` as a prop, so I don't think adding this does anything? 
https://redux-form.com/7.2.2/docs/api/field.md/

Reply:
I didn't put it together because password requires the field "type" or else it doesn't work.

I can change it to have type field and everywhere we call it and don't need it we can leave it blank?

Reply:
@meganhng Can we remove `required` from here? I don't think it's needed, since BE handles incorrect/missing values from the password field, and the `Field` component doesn't do anything with this prop, as seen under the docs. https://redux-form.com/7.2.2/docs/api/field.md/

Reply:
fixed :)